tensorrt_llm/_torch/pyexecutor/model_engine.py

```
@@ -1098,7 +1103,11 @@ def _call_load_weights(self, load_method, weights, weight_mapper):
             load_method(weights)
 
     def _init_max_seq_len(self):
-        inferred_max_seq_len = self.model.infer_max_seq_len()
+        if self.pytorch_backend_config.mm_encoder_only:
+            # TODO: hardcoded for now, need to infer as: max_num_token_per_image * max_num_images (can be given as a parameter or defined same as max_seq_len)
```

Ruff (0.12.2)

Line too long (158 > 120)

(E501)